app/src/main/java/com/teambeme/beme/mypage/adapter/MyWriteAdapter.kt

```diff
@@ -18,6 +18,11 @@ class MyWriteAdapter :
         RecyclerView.ViewHolder(binding.root) {
         fun bind(write: MyWrite) {
             binding.myWrite = write
+            if (write.isSecret) {
+                binding.imgMywriteSecret.setImageResource(R.drawable.ic_secret_on)
+            } else {
+                binding.imgMywriteSecret.setImageResource(R.drawable.ic_secret_off)
+            }
```

Comment:
```suggestion
            android:src="@{myWrite.isSecret ? ic_secret_on : ic_secret_off}"
```

- xml 파일에서 src 설정하는 법 알아봐줘
- 그냥 binding.myWrite 변수만 넣어주고 xml 파일에서 src를 바꿀 수가 있어 한 번 생각해봐줘